frontend/integration-tests/tests/secrets.scenario.ts

```diff
@@ -0,0 +1,75 @@
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+
+const SECRET_NAME = `secret-${testName}`;
+const createWebhookSecretLink = $('#webhook-link');
+
+const secretNameInput = $('#secret-name');
+const revealValuesLink = $('button.btn-link')
+const pre = $$('.co-copy-to-clipboard__text')
+
+const saveButton = $('#save-changes');
+const errorMessage = $('.alert-danger');
```

Comment:
We should probably put the element selectors in a secrets.view.ts file to follow the pattern. Not much here now, but I expect there will be more as we add tests.

Reply:
yeah, was thought that since there is not much would be better to have it in one place. But makes sense to think upfront :+1: